dotnet/samples/Concepts/Memory/VectorStore_VectorSearch_MultiStore_Common.cs

```diff
@@ -47,6 +47,8 @@ public async Task IngestDataAndSearchAsync<TKey>(string collectionName, Func<TKe
         // Upsert the glossary entries into the collection.
         await collection.UpsertAsync(glossaryEntries);
 
+        await Task.Delay(5000); // Add a wait to ensure that indexing completes before we continue.
```

Comment:
Ugh, this kind of thing always sucks :( But yeah, not sure what the alternative is for sample code...